python/src/remotebmi/client/client.py

```diff
@@ -7,14 +7,21 @@
 
 
 class RemoteBmiClient(Bmi):
-    def __init__(self, base_url, timeout=60 * 60 * 24, max_keepalive_connections=0):
+    def __init__(
+        self,
+        base_url,
+        timeout=60 * 60 * 24,
```

Comment:
Is 1 day a reasonable timeout? I guess it needs to be longer than the time it takes to run `.update()` or `update_until()`... but for most users I would not think it makes a difference if it's 1 day or `None`.